src/commands/monthly.ts

```
@@ -77,7 +81,19 @@
 				})),
 				totals: createTotalsObject(totals),
 			};
-			log(JSON.stringify(jsonOutput, null, 2));
+
+			// Process with jq if specified
+			if (ctx.values.jq != null) {
```

ESLint

[error] Unsafe member access .values on an `any` value.

(ts/no-unsafe-member-access)